### PR TITLE
changed the b/g color

### PR DESCRIPTION
**Pull request by lillian-busingye**
changed the background color to aqua

### CODE REVIEW

**Comment by Ddiiro** [comment]
@lillian-busingye please create different folders for your different files. Example `img` `html` `css`.

**Comment by lillian-busingye** [comment]
@Ddiiro this will be worked on thank you

File: style.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
+body {
+    background-color:aqua;
```
Comment (Ddiiro) [inline style.css]: @lillian-busingye kindly change the background color from `aqua` to maybe `gray` as the black text color is hard on the eyes of the clients against the `aqau` background color. WDYT?

Comment (lillian-busingye) [reply style.css]: @Ddiiro this makes sense. thank you

Comment (Ddiiro) [reply style.css]: @lillian-busingye the code LGTM 🎯

**Comment by Ddiiro** [comment]
@lillian-busingye the code LGTM 🎯

**Pull request merged**